server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/status/SnapshotStats.java

```diff
@@ -46,6 +46,11 @@
     SnapshotStats() {
     }
 
+    SnapshotStats(long startTime, long time) {
```

Comment:
I don't think we need another constructor just for this, maybe reuse the existing one?

Reply:
Sure, I guess I hated putting all those `0` but it's shorted this way for sure and horrible regardless :D